utils/resolve.js

```diff
@@ -229,7 +230,7 @@ const erroredContexts = new Set();
  */
 function resolve(p, context) {
   try {
-    return relative(p, context.getPhysicalFilename ? context.getPhysicalFilename() : context.getFilename(), context.settings);
+    return relative(p, getPhysicalFilename(context), context.settings);
```

Comment:
this seems like a noop change, is that correct? (iow, the contextCompat getPhysicalFilename is doing this exact same thing)

Reply:
Almost, but not quite.  The `contextCompat` version would also check `context.filename` before falling back on `context.getFileName()`.  And `context.filename` is the go-forward api whereas `context.getFilename()` is deprecated.

Reply:
aha, thanks :-)